mmpose/models/backbones/shufflenet_v1.py

```diff
@@ -235,7 +235,7 @@ def __init__(self,
 
         self.layers = nn.ModuleList()
         for i, num_blocks in enumerate(self.stage_blocks):
-            first_block = True if i == 0 else False
+            first_block = bool(i == 0)
```

Comment:
`bool` can be omitted

![image](https://user-images.githubusercontent.com/9464825/98687173-6588b980-23a4-11eb-890d-375e0a4a2510.png)

Reply:
Modified.